Formula/amp.rb

```diff
@@ -20,7 +20,11 @@ class Amp < Formula
   uses_from_macos "libiconv"
 
   def install
-    system "cargo", "install", *std_cargo_args
+    if Hardware::CPU.arch == :arm64
+      system "cargo", "install", "--root", prefix, "--path", "."
+    else
+      system "cargo", "install", *std_cargo_args
+    end
```

Comment:
This looks great! 👏

Reply:
OK. Thank you. 
I will fix any other formulae like this amp.rb after checking `brew install --build-from-source amp` is finished.

Reply:
Btw, I think you can actually do
```suggestion
    if Hardware::CPU.arm?
      system "cargo", "install", "--root", prefix, "--path", "."
    else
      system "cargo", "install", *std_cargo_args
    end
```
No need to change your script, though, since these won't be permanent changes.